WEB_ROOT/HTML/ADMIN/ORDER/create-order.html

```diff
@@ -276,8 +276,7 @@ <h1 class="hidden lg:block leading-2 text-3xl font-[inter] text-[#242424]">Metro
             var jobAction = document.getElementById("jobAction").value;
             var jobDescription = document.getElementById("jobDescription").value;
             var jobLink = document.getElementById("jobLink").value;
-            var userId=window.sessionStorage.getItem("userId");
-            console.log("uid",userId);
+            var userId = "58366c01-0b35-4afa-a293-536b711b0a11";
```

Comment:
Do not hardcode values like this.  Pls make user dynamic and come from the logged in user